### DISCUSSION TITLE
Disable multiple devices at once

### DISCUSSION THREAD

**Discussion by BlackRapsbeere** [home-assistant/frontend#9503] [Configuration]
As the architecture for devices especially device_trackers now adds all found devices to HA instead of giving the option to select wanted devices with a ´known-devices.yaml´ e.g. we need to get the UI to disable multiple devices at once.
The simple request:
Add the checkboxes on the devices-tab just as there is on the entities-tab to dis-/enable multiple devices at once.

**Comment by psuter80** [comment]
multi-select is now possible in the devices list but the only two bulk-operations that are possible are to either move the selected devices to an area or add a label. Please add a "Disable" option.

**Reply by TheAndyMac** [reply]
+1 for this
